tensorboard/plugins/profile/README.md

```diff
@@ -97,3 +97,33 @@ The analysis contains three sections:
   * Host-side analysis, which shows you the detailed analysis on the host side,
   including a breakdown of input processing time on the host, and a tabular
   view of details for each input operation.
+
+## Memory Viewer
+
+Memory viewer tool allows you to visualize the peak memory usage for your
+program, and how the memory usage trends over the program's lifetime.
+
+![Memory_Viewer](docs/memory-viewer.png)
```

Comment:
[Optional] Probably not worth the time to change, but `sudo apt-get install pngcrush` is an amazingly good program for files like these.

Reply:
Thanks for the suggestion, it is indeed a great tool.